src/h5web/vis-packs/core/utils.ts

```diff
@@ -235,7 +239,7 @@ export function getCanvasScale(
  * So we implement our own, simplified version of `d3.ticks` that always outputs integer values.
  */
 export function getIntegerTicks(domain: Domain, count: number): number[] {
-  const [min, max] = domain;
+  const [min, max] = domain[1] < domain[0] ? [domain[1], domain[0]] : domain;
```

Comment:
The method was not returning any ticks if the domain was inverted.

Reply:
Might be easier to do `const min = Math.min(...domain)` ?

Reply:
Feels a bit odd when `domain` only contains two values but why not.

Reply:
meh, up to you ^^